incubator/hnc/api/v1alpha2/hierarchy_types.go

```diff
@@ -31,11 +31,15 @@ const (
 
 // Constants for labels and annotations
 const (
-	MetaGroup                = "hnc.x-k8s.io"
-	LabelInheritedFrom       = MetaGroup + "/inheritedFrom"
-	FinalizerHasSubnamespace = MetaGroup + "/hasSubnamespace"
-	LabelTreeDepthSuffix     = ".tree." + MetaGroup + "/depth"
-	AnnotationManagedBy      = MetaGroup + "/managedBy"
+	MetaGroup                 = "hnc.x-k8s.io"
+	LabelInheritedFrom        = MetaGroup + "/inheritedFrom"
+	FinalizerHasSubnamespace  = MetaGroup + "/hasSubnamespace"
+	LabelTreeDepthSuffix      = ".tree." + MetaGroup + "/depth"
+	AnnotationManagedBy       = MetaGroup + "/managedBy"
+	AnnotationPropagatePrefix = "propagate." + MetaGroup
+
+	AnnotationSelector     = AnnotationPropagatePrefix + "/select"
+	AnnotationTreeSelector = AnnotationPropagatePrefix + "/treeSelect"
```

Comment:
Is `AnnotationTreeSelector` not used anywhere in this PR yet?

Reply:
yes

Reply:
If it's not used, it should probably be removed. It's best not to add stuff before you need it.